ansible/roles/ocp4-workload-er-demo/tasks/remove_workload.yml

```diff
@@ -1,15 +1,15 @@
 ---
 
-- name: "Uninstall ER-Demo using inventory file at {{ inventory_hosts_file }}"
+- name: "Uninstall ER-Demo using inventory file at {{ ocp4_workload_er_demo_inventory_hosts_file }}"
   shell: |
-          ansible-playbook -i "{{ inventory_hosts_file }}" \
+          ansible-playbook -i "{{ ocp4_workload_er_demo_inventory_hosts_file }}" \
```

Comment:
This is still making me uneasy. How do we check it deployed successfully? How do we notice if it failed?

Can this be converted into proper tasks to be included rather than calling the playbook?